### PR TITLE
chore(radix-docs): improve the home page for all screen sizes

### PR DESCRIPTION
**Pull request by KamilEmeleev**
Improve the home page for all screen sizes

### CODE REVIEW

File: apps/radix-docs/src/components/layouts/DemosHomePage.astro [hunk 1]
```diff
@@ -1,8 +1,30 @@
...
-import DemoContainer from './DemoContainer.astro';
+import DemoNoopContainer from '@/components/layouts/DemoNoopContainer.astro';
 ---
 
-<DemoContainer title="accordion" componentName="accordion" componentFile="accordion-demo" />
-<DemoContainer title="avatar" componentName="avatar" componentFile="avatar-demo" />
-<DemoContainer title="collapsible" componentName="collapsible" componentFile="collapsible-demo" />
-<DemoContainer title="dropdown menu" componentName="dropdown-menu" componentFile="dropdown-menu-demo" />
+<div class="grid gap-4 sm:grid-cols-1 md:grid-cols-2 md:p-4 xl:grid-cols-3">
+    <!-- Левая колонка: два блока друг под другом -->
```
Comment (artembelik) [inline apps/radix-docs/src/components/layouts/DemosHomePage.astro:6]:
```suggestion
```
we prefer to use eng :)

File: apps/radix-docs/src/demos/primitives/collapsible/collapsible-demo.ts [hunk 2]
```diff
@@ -19,9 +19,9 @@ import { LucideAngularModule, UnfoldVertical, X } from 'lucide-angular';
     ],
     template: `
         <div class="CollapsibleRoot" #collapsibleRoot="collapsibleRoot" [open]="true" rdxCollapsibleRoot>
-            <div style="display: flex; align-items: center; justify-content: space-between;">
+            <div style="display: flex; align-items: center; justify-content: space-between; gap: 1em;">
                 <span class="Text" style="color: white;">&#64;peduarte starred 3 repositories</span>
-                <button class="IconButton" rdxCollapsibleTrigger>
+                <button class="IconButton" style="flex-shrink: 0;" rdxCollapsibleTrigger>
```
Comment (artembelik) [inline apps/radix-docs/src/demos/primitives/collapsible/collapsible-demo.ts:24]: use [flex](https://tailwindcss.com/docs/flex-shrink) selectors, like `shrink-0` etc

File: apps/radix-docs/src/components/layouts/DemosHomePage.astro (hunk 1 above)
Comment (KamilEmeleev) [reply apps/radix-docs/src/components/layouts/DemosHomePage.astro]: thanks, I have fixed it

File: apps/radix-docs/src/demos/primitives/collapsible/collapsible-demo.ts (hunk 2 above)
Comment (KamilEmeleev) [reply apps/radix-docs/src/demos/primitives/collapsible/collapsible-demo.ts:24]: it seem to me, we don't use tailiwind in the component examples

**Review by artembelik — APPROVED** [review]

**Review by pimenovoleg — APPROVED** [review]

**Pull request merged**